badge_server/main.py

```diff
@@ -42,12 +42,13 @@
 
 import utils as badge_utils
 from compatibility_lib import utils as compat_utils
+from compatibility_lib.compatibility_store import CompatibilityResult
```

Comment:
You can only import modules (see http://google.github.io/styleguide/pyguide.html#22-imports)

Reply:
done